packages/e2e-test-utils-playwright/src/editor/toggle-fixed-toolbar.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Toggles the fixed toolbar option.
+ *
+ * @param this
+ * @param setIsFixedToolbar Boolean value true/false for on/off.
+ */
+export async function toggleFixedToolbar(
```

Comment:
Oh oops, I mean the util name, not the parameter name. `isFixed` is actually a more readable parameter name too.